### ISSUE TITLE
[Accessibility] Keyboard shortcuts documents in PDF form are not accessible

### ISSUE THREAD

**Issue by tigoe** [Microsoft/vscode#59969]
- VSCode Version: 1.27.2
- OS Version: MacOS 10.12.6

Steps to Reproduce:

1. open https://code.visualstudio.com/shortcuts/keyboard-shortcuts-macos.pdf
2. turn on VoiceOver on Mac (command-F5)

This will quickly reveal that the PDF is incomprehensible from the VoiceOver screen reader in a browser (tested with Chrome, Safari, and Firefox).  Further steps to try to read it:

3. download PDF, open in Acrobat Reader DC or Preview
4. turn on VoiceOver on Mac (command-F5)

Again, the document is incomprehensible since the layout is not ordered. You have to read through the column of keystrokes, for example, before you get to the column of what they do.

Given the effort to make VS Code accessible, it would be really good if the documentation for keyboard use were equally as accessible. 

Does this issue occur when all extensions are disabled?: Not relevant
